python/cuml/linear_model/linear_regression.pyx

```diff
@@ -216,7 +218,10 @@ class LinearRegression:
         cdef uintptr_t coef_ptr = self._get_column_ptr(self.coef_)
 
         cdef float c_intercept1
+        print(c_intercept1)
```

Comment:
This probably was left by accident, but same comment as the other print statements.

Reply:
Sorry, yes I left the print statements in the code by accident